ocaml/idl/datamodel_repository.ml

```diff
@@ -73,6 +82,38 @@ let introduce =
     ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
     ()
 
+let introduce_bundle =
+  call ~name:"introduce_bundle" ~in_oss_since:None
+    ~lifecycle:[(Published, "1.301.0", "")]
+    ~doc:"Add the configuration for a new bundle repository"
+    ~versioned_params:
+      [
+        {
+          param_type= String
+        ; param_name= "name_label"
+        ; param_doc= "The name of the repository"
+        ; param_release= numbered_release "1.301.0"
```

Comment:
Put (here and below) something like `24.19.1-next` for the time being, so we can find it before merging to master.